projects/ansible/collections/dev_guidelines/releasing_collections_without_release_branches.rst

```diff
@@ -0,0 +1,93 @@
+**********************************************
+Releasing collections without Release Branches
+**********************************************
+
+If you have any question when reading this manual, ask the community in the ``#ansible-community`` IRC channel or create an issue in the `community-docs <https://github.com/ansible/community-docs>`_ repository.
+
+This manual assumes that publishing the collection is done via `Zuul <https://github.com/ansible/project-config>`_ and that `antsibull-changelog <https://github.com/ansible-community/antsibull-changelog>`_ is used for the changelog.
+
+Since no release branches are used, the manual does not distinguish between releasing a major, minor, or patch version.
+
+1. First, examine the collection if there are merged changes to release.
+
+2. According to the changes made, choose an appropriate release version number. Keep in mind that the collections must follow the `semantic versioning <https://semver.org/>`_ rules. Refer to the `Releasing without release branches <releasing_collections_without_release_branches.rst>`_ for details.
```

Comment:
The file is currently refering to itself for details.

Reply:
Ohh, will fix, thanks